docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -151,6 +151,30 @@ installation directory to the `Path` user environment variable.
 </TabItem>
 </Tabs>
 
+## Telemetry
+
+(!docs/pages/includes/teleport-connect-telemetry.mdx!)
+
+### Disabling telemetry
+
+If you initially agreed to share telemetry data, but now want to opt out, you need to update the `app_config.json` file:
```

Comment:
Does the app need to be closed first? I don't remember if we dump the current config to the file just like we do with app_state.json.

Reply:
No, it doesn't. The config file is updated only on change.